std/regex/package.d

```diff
@@ -136,7 +136,7 @@ $(TR $(TD Objects) $(TD
     $(REG_ROW \W, Matches any non-word character.)
     $(REG_ROW \s, Matches whitespace, same as \p{White_Space}.)
     $(REG_ROW \S, Matches any character except those recognized as $(I \s ). )
-    $(REG_ROW \\, Matches \ character. )
+    $(REG_ROW \\\\, Matches \ character. )
```

Comment:
It would have been nice to make it so that it would be possible to write DDoc that would render the same with or without `-preview=markdown`, but I guess such changes are unavoidable in practice. Main concern is tools which consume DDoc directly, such as DDox, or editors/IDEs which parse DDoc to display formatted documentation in their UI.